src/main/java/com/company/client/bot/MyBot.java

```diff
@@ -47,20 +45,20 @@ public MyBot(String botToken) {
     @SneakyThrows
     @Override
     public void onUpdateReceived(Update update) {
+
         if (update.hasMessage()) {
             Message message = update.getMessage();
             Long chatId = update.getMessage().getChatId();
 
             Optional<User> optionalUser = userService.getByChatId(chatId);
             String username = update.getMessage().getChat().getUserName();
 
-            User user = optionalUser.orElse(User.builder()
+            user = optionalUser.orElse(User.builder()
```

Comment:
please create UserFactor class in the factory package and create method which returns object of user